bimsb/packages/tainted.scm

```diff
@@ -188,6 +188,8 @@ deviation (SD) plots, coefficient of variation (CV) plots.")
                (for-each
```

Comment:
This line wasn't changed by you, but maybe we can fix this while you're at it.  `substitute*` takes a list of files, so we don't need to `for-each` a `lambda` binding `script` over a list of files --- we can simply write `(substitute* '("src/..." "..." "...")...)`.

Reply:
Sure. I'll have a look at it.

Reply:
I gave it a shot and it seems I got it right. At least I can still build locally and run the tutorial just fine.
Thanks for the help simplifying this.